songr/src/main/java/com/lab11java401d3/songr/AlbumController.java

```diff
@@ -0,0 +1,39 @@
+package com.lab11java401d3.songr;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController

```

Comment:
Restcontrollers automatically make the response body a string. In this lab, you'll need to render the info on the webpage using thymeleaf. I recommend using @Controller instead.